lib/vet360/stats.rb

```diff
@@ -35,6 +35,15 @@ def increment_transaction_results(response, bucket1 = 'posts_and_puts')
         increment(bucket1, bucket_for(status))
       end
 
+      # Increments the associated StatsD bucket with the passed in exception error key.
+      #
+      # @param key [String] A Vet360 exception key from the locales/exceptions file
+      #   For example, 'VET360_ADDR133'.
+      #
+      def increment_exception(key)
+        StatsD.increment("#{Vet360::Service::STATSD_KEY_PREFIX}.exceptions", tags: ["exception:#{key.downcase}"])
```

Comment:
I didn't notice this before, but I see that we are downcase-ing on the increment, but not on the initialization. Is that because they pull from different sources? Or should we do the same in both places? (Prometheus I think will need them to be consistent, whichever route we go)

Reply:
@wyattwalter - Yeah, we are purposefully downcasing in both places, the initialization and the incrementing.  This array that is being iterated over actually returns downcased Vet360 exception keys:

https://github.com/department-of-veterans-affairs/vets-api/blob/5f6d068deb9fe1d431c66a64a37eb57597d0d1a4/config/initializers/statsd.rb#L66

i.e. 

```ruby
=> ["vet360_502", "vet360_addr101", "vet360_addr102", ...]
```

Thanks

Reply:
I see, the keys are already run through `downcase` when loaded by `known_keys` 👍